pyproject.toml

```
@@ -1,20 +1,24 @@
-[tool.poetry]
+[project]
 name = "exasol-schemas"
+description = "Definitions for Exasol-provided schemas (XML, JSON, etc.)"
+version = "0.1.0"
```

```suggestion
version = "1.0.0"
```
Changelog says 1.0.0